src/auth-service/utils/inquire.js

```diff
@@ -18,23 +18,23 @@ const inquire = {
     try {
       let { fullName, email, message, category, tenant } = inquire;
 
-      await validationsUtil.checkEmailExistenceUsingKickbox(email, (value) => {
-        if (value.success === false) {
-          const errors = value.errors ? value.errors : "";
-          logObject("the validation checks results", {
-            success: false,
-            message: value.message,
-            errors,
-            status: value.status,
-          });
-          // callback({
-          //   success: false,
-          //   message: value.message,
-          //   errors,
-          //   status: value.status,
-          // });
-        }
-      });
+      // await validationsUtil.checkEmailExistenceUsingKickbox(email, (value) => {
```

Comment:
just commenting (disabling) all sections which connect to Kickbox - our credit got done so the tool is not functional right now.